_data/students.yml

```diff
@@ -160,3 +160,10 @@
 - name: Yash Garg
   github: Yash-Garg
   image: yashg.png
+  
+- name: 三銃士
+  github: namphong124563
+  image: namphong124563.jpeg
+  facebook: namphong124563
```

Comment:
Please remove these extra lines. Only give your name, image and github link.

Reply:
Finished.